src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/EntityViewScopeWidget.java

```diff
@@ -167,25 +167,16 @@ public void onFailure(Throwable caught) {
 	}
 
 	@Override
-	public void onEdit() {
-		List<Reference> references = new ArrayList<>();
+	public void onEditScopeAndMask() {
+		List<Reference> references = getReferencesFromIdList(currentView.getScopeIds());
 		// configure edit list, and show modal
-		if (currentView instanceof EntityView) {
-			references = getReferencesFromIdList(((EntityView) currentView).getScopeIds());
-		} else if (currentView instanceof Dataset) {
-			references = getReferencesFromDatasetItems(((Dataset) currentView).getItems());
-		}
 		editScopeWidget.configure(references, true, tableType);
 
-		if (TableType.table.equals(tableType) || TableType.projects.equals(tableType) || TableType.dataset.equals(tableType)) {
-			view.setViewTypeOptionsVisible(false);
```

Comment:
This path modified the dialog, but is now unnecessary because we will never show the dialog if 'viewTypeOptions' (the view mask editor) shouldn't be visible.